<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmQRcodeIdentifyClass</class>
 <widget class="QDialog" name="frmQRcodeIdentifyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>655</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>二维码识别</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #bcc2bc;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <widget class="QGroupBox" name="groupC1">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>18</y>
         <width>427</width>
         <height>73</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>图像对象</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>42</y>
          <width>79</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string>输入图像：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtLinkImage">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>39</y>
          <width>255</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLinkImage">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>39</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>链接图像</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/link.png</normaloff>:/resource/link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelLinkImage">
        <property name="geometry">
         <rect>
          <x>394</x>
          <y>39</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>删除图像链接</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(226, 142, 162, 100);}
QPushButton:pressed{background-color: rgba(226, 172, 162, 200);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/del_link.png</normaloff>:/resource/del_link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupC_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>106</y>
         <width>427</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>二维码类型</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>39</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string>支持的二维码类型：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboMode">
        <property name="geometry">
         <rect>
          <x>166</x>
          <y>34</y>
          <width>120</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>188</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 2px;    
    color: rgb(0, 0, 200);
	background-color: rgba(170, 170, 127,100);
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px; 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/resource/down.png);
} 
QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>QRCode</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupC3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>192</y>
         <width>427</width>
         <height>151</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>二维码参数</string>
       </property>
       <widget class="QGroupBox" name="groupC4">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>34</y>
          <width>403</width>
          <height>67</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #acbbac;
color: rgb(0, 0, 127);</string>
        </property>
        <property name="title">
         <string>图像相乘参数</string>
        </property>
        <widget class="QDoubleSpinBox" name="spinMulValue">
         <property name="geometry">
          <rect>
           <x>97</x>
           <y>32</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{color: rgb(85, 0, 255);background-color: rgb(180, 180, 210);}
QDoubleSpinBox::up-button{
    border:transparent;
}
QDoubleSpinBox::down-button{
    border:transparent;
}
QDoubleSpinBox::up-arrow{
    image:url(:/resource/up.png);
}
QDoubleSpinBox::down-arrow{
    image:url(:/resource/down.png);
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>21.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.003000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_52">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>36</y>
           <width>83</width>
           <height>19</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
         </property>
         <property name="text">
          <string>相乘系数：</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnTestImage">
        <property name="geometry">
         <rect>
          <x>334</x>
          <y>120</y>
          <width>83</width>
          <height>32</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ background-color: rgba(9, 42, 87, 200);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 5px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(9, 62, 87);}
QPushButton:pressed{background-color: rgb(9, 82, 87);}</string>
        </property>
        <property name="text">
         <string>测试图像</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>285</x>
          <y>118</y>
          <width>43</width>
          <height>35</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sub_res.qrc">:/resource/a_right.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupC6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>431</width>
         <height>371</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>识别内容</string>
       </property>
       <widget class="QTextEdit" name="txtMsg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>419</width>
          <height>321</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit         
{
    background-color: rgba(180, 180, 215,120);    
    gridline-color: rgba(220, 220, 255,255);
    color: rgba(20, 20, 20,200);
}
QScrollBar:vertical{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:vertical{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:vertical:hover{
		background:#d0d0d0;}
QScrollBar::add-line:vertical{
		background:url(:/resource/down.png) center no-repeat;}
QScrollBar::sub-line:vertical{
		background:url(:/resource/up.png) center no-repeat;}
QScrollBar:horizontal{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:horizontal{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:horizontal:hover{
		background:#d0d0d0;}
QScrollBar::add-line:horizontal{
		background:url(:/resource/right.png) center no-repeat;}
QScrollBar::sub-line:horizontal{
		background:url(:/resource/left.png) center no-repeat;}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>单元描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDescribe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>357</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ background-color: rgb(85, 135, 127);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(85, 155, 127);}
QPushButton:pressed{background-color: rgb(85, 175, 127);}</string>
       </property>
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>6</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="imageLayout">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sub_res.qrc"/>
  <include location="../../ImageProcess/ImageView/sub_res.qrc"/>
  <include location="../BarcodeIdentify/sub_res.qrc"/>
 </resources>
 <connections/>
</ui>
